### ISSUE TITLE
Add run information in DL1DataReader

### ISSUE THREAD

**Issue by mikael10j** [cta-observatory/dl1-data-handler#64]
To fully benefit from Gammaboard functionalities, we need to keep track of:
- num_showers
- shower_reuse
- spectral_index
- max_scatter_range
- energy_range_max
- energy_range_min
- min_alt
- max_alt
- mc_energy of triggered events

of every runs in the DL1DataReader

**Comment by TjarkMiener** [comment]
Hi @mikael10j! how should we do this? Most of the information is stored in the metadata from the file. We could read it with `f.root._v_attrs. num_showers` per run and store a class dict with the run number as key in the reader.

**Comment by mikael10j** [comment]
hi Tjark,

I handle it with a MCHeaderContainer:

```python
        self.run_config = {}

        with tables.File(hdf5_file_path, 'r') as hdf5_file:
            # Load run config
            mcheader = {}
            for attr in hdf5_file.root._v_attrs._f_list('user'):
                if attr in ['ctapipe_version', 'dl1_data_handler_version', 'runlist']:
                    self.run_config[attr] = hdf5_file.root._v_attrs[attr]
                else:
                    mcheader[attr] = hdf5_file.root._v_attrs[attr]
                self.run_config[attr] = hdf5_file.root._v_attrs[attr]
            self.run_config['mcheader'] = MCHeaderContainer(**mcheader)
```
It is then very easy to add this to the output files produced at inference time.

**Comment by TjarkMiener** [comment]
Solved by v0.10.0.

You can retrieve the simulation info needed for pyIRF:
`from dl1_data_handler.reader import DL1DataReaderSTAGE1`
`reader = DL1DataReaderSTAGE1(**config['Data'])`
`reader.pyIRFSimulatedEventsInfo`
